py/legacypipe/reference.py

```diff
@@ -343,6 +362,51 @@ def read_large_galaxies(survey, targetwcs):
     galaxies.rename('lslga_id', 'ref_id')
     galaxies.ref_cat = np.array([refcat] * len(galaxies))
     galaxies.islargegalaxy = np.ones(len(galaxies), bool)
+    galaxies.freezeparams = np.zeros(len(galaxies), bool)
+    
+    galaxies.sources = np.array([None] * len(galaxies))
+
+    ## FIXME
+    if 'sersic' in galaxies.get_columns():
+        from legacypipe.catalog import fits_reverse_typemap
+        from tractor.wcs import RaDecPos
+        from tractor import NanoMaggies
+        from tractor.ellipses import EllipseE
+        from tractor.galaxy import DevGalaxy, ExpGalaxy
+        from tractor.sersic import SersicGalaxy, SersicIndex
+
+        for i,g in enumerate(galaxies):
+            try:
+                typ = fits_reverse_typemap[g.type.strip()]
+                pos = RaDecPos(g.ra, g.dec)
+                fluxes = dict([(band, g.get('flux_%s' % band)) for band in bands])
+                bright = NanoMaggies(order=bands, **fluxes)
+                if issubclass(typ, (DevGalaxy, ExpGalaxy, SersicGalaxy)):
+                    shape = EllipseE(g.shape_r, g.shape_e1, g.shape_e2)
+
+                if issubclass(typ, (DevGalaxy, ExpGalaxy)):
+                    src = typ(pos, bright, shape)
+                    print('Created', src)
+                elif issubclass(typ, (SersicGalaxy)):
+                    sersic = SersicIndex(g.sersic)
+                    src = typ(pos, bright, shape, sersic)
+                    print('Created', src)
+                else:
+                    print('Unknown type', typ)
+                galaxies.sources[i] = src
+                galaxies.freezeparams[i] = True
+
+            except:
+                import traceback
+                print('Failed to create Tractor source for LSLGA entry:', traceback.print_exc())
+
+
+    keep_columns = ['ra', 'dec', 'radius', 'mag', 'ref_cat','ref_id', 'ba', 'pa', 'sources',
+                    'islargegalaxy', 'freezeparams']
+    for c in galaxies.get_columns():
+        if not c in keep_columns:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [test for membership should be 'not in'](https://app.codacy.com/manual/dstndstn/legacypipe/pullRequest?prid=4930373)